src/mlj_catboostclassifier.jl

```diff
@@ -103,8 +107,7 @@ function MMI.predict(mlj_model::CatBoostClassifier, fitresult, X_pool)
         n = pyconvert(Int, X_pool.shape[0])
         classes = [fitresult.single_class]
         probs = ones(n, 1)
-        pool = MMI.categorical([fitresult.y_first])
-        return MMI.UnivariateFinite(classes, probs; pool=pool)
+        return MMI.UnivariateFinite(classes, probs; pool=fitresult.y_first)
     end
```

Comment:
Thanks @tylerjthomas9  for your work on this issue. 
Something doesn't look right to me. The idea is that the complete categorical pool should be passed to `UnivariateFinite` even if only one class is actually manifest in the training. This will happen if the first argument of `UnivariateFinite` is a categorical vector with complete pool (even if it does not include all classes as manifest elements) or if something with the complete pool is passed to `pool=...` kwarg (could be a single `CategoricalValue`). You're saying that `y_first` may not be a `CategoricalValue`. But if that is true, it won't contain the complete class pool, which you must get from somewhere else (something recorded in `fitresult`). 

I realize this can be confusing, and am happy to clarify further.